packages/client-gear-playground/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "client-gear-playground",
+  "version": "1.0.0",
+  "description": "A place to test and development kununu components",
```

Comment:
A place to test and develop client-gear? It seems the playground will not be just for components after all.